plugins/CoreHome/Columns/Metrics/ConversionRate.php

```diff
@@ -43,8 +44,28 @@ public function format($value, Formatter $formatter)
 
     public function compute(Row $row)
     {
-        $nbVisitsConverted = $this->getMetric($row, 'nb_visits_converted');
         $nbVisits = $this->getMetric($row, 'nb_visits');
+        $nbVisitsConverted = $this->getMetric($row, 'nb_visits_converted');
+        if ($nbVisitsConverted === false) {
+            $goals = $this->getMetric($row, 'goals');
+
+            if (!empty($goals)) {
+                $nbVisitsConverted = 0;
+                foreach ($goals as $idGoal => $values) {
+                    if ($idGoal === 0
+                        || $idGoal === '0'
```

Comment:
is there maybe a CONST we could reference instead of hard coding the value?

Reply:
Can do